api-server/server/boot/user.js

```diff
@@ -246,8 +247,10 @@ function createPostReportUserProfile(app) {
         }
 
         return res.json({
-          typer: 'info',
-          message: `A report was sent to the team with ${user.email} in copy.`
```

Comment:
Not that it makes much difference, but the `translation.json` message does not have a period at the end of the sentence like this does.

On a side note, I noticed a couple others that either added a period to the end that was not there or removed a period that was there in the original message.  Again, not a big deal, but after looking through all the messages (which is easier now that they are all in the same file), I notice we have not been consistent with using/not using periods at the end of the messages.  My personal preference would be to have a period if the message is a full sentence.

Reply:
I may have added and removed some periods here and there to try and make things consistent. But some messages/text seem to need them and some don't - it seems like a judgment call. Not sure if we should try and come up with a standard. I'm open to discussing it.